smieci/react/colorPicker/src/App.tsx

```diff
@@ -1,8 +1,28 @@
 import ColorPicker from "./ColorPicker"
+import Home from "./Home"
+import Login from "./Login"
+import Register from "./Register"
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom"
+import { useState } from "react"
 
 function App() {
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
```

Comment:
1) use axios instead of fetch

1.5) create some endpoint that you will actually use. idk, some small one: for example after registration, when i click button my email is fetched from serwer and displayed. (only authorized user can see his own email)

2) Change in /colorpicker/security/config/JwtService.java

the JWT expiration to 30 sec. 

So after 30sec of using JWT it will stop working . How do you think we can handle it?

3) yea i will tell you. the easiest solution is to  set axios globall [error handler](https://stackoverflow.com/questions/48990632/how-to-manage-axios-errors-globally-or-from-one-point)  . If you are getting 401 not authorized code from serwer redirect to login page.